packages/media-sound/flac123/flac123-0.0.12.exheres-0

```diff
@@ -1,21 +1,16 @@
 # Copyright 2009 Ali Polatel <alip@exherbo.org>
 # Distributed under the terms of the GNU General Public License v2
 
-require sourceforge [ project=flac-tools suffix=tar.gz ]
+require sourceforge [ project=flac-tools suffix=tar.gz pnv=${PNV}-release ]
+require autotools [ supported_autoconf=[ 2.5 ] supported_automake=[ 1.16 1.15 1.14 ] ]
```

Comment:
for autoconf there's 2.7 and just use only latest if adding it just now, so I'd suggest:
`require autotools [ supported_autoconf=[ 2.7 ] supported_automake=[ 1.16 ] ]`